packages/firestore/test/unit/util/bundle.test.ts

```diff
@@ -241,7 +241,7 @@ function genericBundleReadingTests(bytesPerRead: number): void {
 
       await expect(
         generateBundleAndParse('{metadata: "no length prefix"}', bytesPerRead)
-      ).to.be.rejectedWith('Unexpected end of JSON input');
+      ).to.be.rejectedWith(/(Unexpected end of )(?=.*\JSON\b).*/gi);
```

Comment:
I don't think that you need the `\` before the "J", I plugged it into regexr.com and I think that just escapes the "J" which isn't needed.